pymake/makefile

```diff
@@ -43,108 +43,114 @@ ${SOURCEDIR14}
 
 # Define the Fortran compile flags
 FC = gfortran
-FFLAGS = -O2 -fbacktrace -ffpe-summary=overflow -ffpe-trap=overflow,zero,invalid 
+FFLAGS = -O2 -ffpe-summary=overflow -ffpe-trap=overflow,zero,invalid -fbacktrace --coverage
```

Comment:
--coverage was added on purpose?

Reply:
Good catch!  I was testing with the fortran code coverage and forgot to remove this.